x/authz/types/codec.go

```diff
@@ -11,12 +10,6 @@ import (
 
 // RegisterInterfaces registers the interfaces types with the interface registry
 func RegisterInterfaces(registry types.InterfaceRegistry) {
-	registry.RegisterImplementations((*sdk.MsgRequest)(nil),
```

Comment:
do these get removed from all modules or just the ones in these modules?

Reply:
Ah you're right, these messages should still implement the (new) `sdk.Msg` interface, like all messages.